doc/source/whatsnew/v1.3.0.rst

```diff
@@ -796,6 +796,7 @@ I/O
 - Bug in :func:`read_excel` dropping empty values from single-column spreadsheets (:issue:`39808`)
 - Bug in :meth:`DataFrame.to_string` misplacing the truncation column when ``index=False`` (:issue:`40907`)
 - Bug in :func:`read_orc` always raising ``AttributeError`` (:issue:`40918`)
+- Bug in :func:`read_clipboard` when copying from excel and the first column contains null values (:issue:`41108`)
```

Comment:
Can you include what happens in the bug, something like "would incorrectly shift values between columns"

Reply:
changed it to this, would that work?

```diff
+ - Bug in :func:`read_clipboard` copying from an excel file shifts values into the wrong column if there are null values in first column (:issue:`41108`)
```